mwclient/client.py

```diff
@@ -121,7 +127,7 @@ def __init__(self, host, path='/w/', ext='.php', pool=None, retry_timeout=30,
                     raise errors.OAuthAuthorizationError(e.code, e.info)
 
                 # Private wiki, do init after login
-                if e.args[0] not in (u'unknown_action', u'readapidenied'):
+                if e.args[0] not in {u'unknown_action', u'readapidenied'}:
```

Comment:
__Ew__!

Turns out 2.6 doesn't know the dict literal. Workaround would be using `set(…)`

Are there any plans on dropping 2.6 support?